packages/core/src/combinator/combine.ts

```diff
@@ -3,43 +3,48 @@
 import { map } from './transform'
 import { empty, containsCanonicalEmpty } from '../source/empty'
 import Pipe from '../sink/Pipe'
-import IndexSink from '../sink/IndexSink'
+import IndexSink, { IndexedValue } from '../sink/IndexSink' // eslint-disable-line no-unused-vars
```

Comment:
This is because of `standardjs`. Please see https://github.com/mostjs/hold/pull/36#discussion_r301988875

Reply:
Yes, but I don’t understand _why_ `no-unused-vars` should be disabled for that particular line. Which variables declared on that line are not used anywhere in the code? And if any, why are they announced when not used?

Reply:
This happens because of imported _type_ `IndexedValue`, not a _value_.
The issues: https://github.com/eslint/typescript-eslint-parser/issues/457, https://github.com/bradzacher/eslint-plugin-typescript/issues/90, https://github.com/bradzacher/eslint-plugin-typescript/issues/112

Reply:
How about moving to plain `eslint` bin with `eslint-plugin-standard`? This should fix the issue (I guess) and should still fix the code with `--fix` flag.

Reply:
I think this is a known issue with the typescript eslints plugins.  I remember seeing an issue for it ... maybe this one: https://github.com/typescript-eslint/typescript-eslint/issues/363 but not sure.

Reply:
I managed to solve this using `eslint` and https://github.com/standard/eslint-config-standard-with-typescript. Will need some type to update the code.

Reply:
Awesome!  Thanks for finding a solution.